### PR TITLE
[jog_arm] SRDF velocity and acceleration limit enforcement

### PR DESCRIPTION
**Pull request by AndyZe**
Read velocity and acceleration limits from the SRDF and enforce them

### CODE REVIEW

**Comment by AndyZe** [comment]
The problem with the tests (I think) is that it's a fairly large integration test, and it includes several sub-tests that exceed the 60s timeout. There are some small improvements I can make to get is passing for now.

**Comment by henningkayser** [comment]
@AndyZe Please rebase

File: moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_calcs.h [hunk 1]
```diff
@@ -142,7 +143,7 @@ class JogCalcs
 
   robot_state::RobotStatePtr kinematic_state_;
 
-  sensor_msgs::JointState joint_state_, original_joint_state_;
+  sensor_msgs::JointState incoming_joint_state_, internal_joint_state_, original_joint_state_;
```
Comment (felixvd) [inline moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_calcs.h]: Does the `internal_joint_state_` correspond to a "current" joint state?

Comment (AndyZe) [reply moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_calcs.h]: What I mean is that it's used in calculations, it shouldn't be relied on to be accurate. Would there be a better way to say that?

I'm gonna try to get this list down to 2. It seems like incoming_joint_state_ and original_joint_state_ should be the same thing.

**Comment by AndyZe** [comment]
The test that was added (test_vel_accel_limits.py) will make more sense when I can check for specific error codes. For now, it's just a placeholder.

File: moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_calcs.h (hunk 1 above)
Comment (AndyZe) [reply moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_calcs.h]: It turns out that `original_joint_state_` is a little bit different from `incoming_joint_state_` so I left a comment to explain it. `incoming_joint_state_` has every joint, but passive joints might get dropped from `original_joint_state_`

**Pull request merged**